doc/guides/dts/howtos.rst

```diff
@@ -202,7 +202,21 @@ Set devicetree overlays
 Devicetree overlays are explained in :ref:`devicetree-intro`. The CMake
 variable :makevar:`DTC_OVERLAY_FILE` contains a space- or semicolon-separated
 list of overlay files to use. If :makevar:`DTC_OVERLAY_FILE` specifies multiple
-files, they are included in that order by the C preprocessor.
+files, they are included in that order by the C preprocessor.  The absolute
+path to a file in a module will be generated by referring to it in your
+project's :file:`CMakeLists.txt` as:
+
+.. code-block:: cmake
+
+   set(DTC_OVERLAY_FILE "\${ZEPHYR_MODULENAME_MODULE_DIR}/<module-relative-pathname>")
+
+   find_package(Zephyr)
+
+or from the shell as:
+
+.. code-block:: shell
+
+   cmake -DDTC_OVERLAY_FILE="\${ZEPHYR_MODULENAME_MODULE_DIR}/<module-relative-pathname>"
```

Comment:
Related to the comment of the rendered output.

How about just a short sentence on this, for example like:
```suggestion
files, they are included in that order by the C preprocessor. A file in a Zephyr
module can be referred by escaping the Zephyr module dir variable like `\${ZEPHYR_<module>_MODULE_DIR}/<path-to>/dts.overlay` when setting the DTC_OVERLAY_FILE variable.
```